plugins/filter_aws/aws.c

```diff
@@ -1048,6 +1114,12 @@ static struct flb_config_map config_map[] = {
      "if both tags_include and tags_exclude are specified, configuration is invalid"
      " and plugin fails"
     },
+    {
+     FLB_CONFIG_MAP_INT, "tags_retry_interval_s", "5",
```

Comment:
I think we should pick a default that's higher than 5? That's 12 calls per minute, which is a lot.

Reply:
I've changed the default value to 300 (5m).